runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/GaugeData.java

```diff
@@ -57,7 +58,7 @@ public GaugeResult extractResult() {
   public static class EmptyGaugeData extends GaugeData {
 
     private static final EmptyGaugeData INSTANCE = new EmptyGaugeData();
-    private static final Instant EPOCH = new Instant(0);
+    private static final Instant EPOCH = new Instant(GlobalWindow.TIMESTAMP_MIN_VALUE);
```

Comment:
Why this change?

Reply:
I think this is a bug because the empty gauge will only be overridden with a timestamp > 0, but any time stamp > min timestamp should override it.

Reply:
Better to rename the constant in this case?

Reply:
We can rename it in a follow-up if you don't mind.